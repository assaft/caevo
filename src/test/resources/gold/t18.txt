He kicked the ball, and it rose into the air.