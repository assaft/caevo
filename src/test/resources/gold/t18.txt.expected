ei1,kicked
ei2,rose
l1,ei1,ei2,BEFORE